README.md

```diff
@@ -103,3 +103,60 @@ Produce code coverage report
     $ cabal install --enable-tests --enable-coverage
     $ cabal test
     $ firefox ./dist/hpc/vanilla/tix/*/hpc_index.html
+
+OSTree cheat sheet
+==================
+
+After importing RPMs (via the `import` executable) the results are a
+`metadata.db` SQL database and a `cs.repo` directory containing an
+OSTree repository. Here are a few quick examples how to work with `ostree`:
+
+
+To see what references aka "branches" are in the repo:
+
+    ostree --repo=cs.repo/ refs
+    master
+
+
+To see the commit log (use less, ostree doesn't paginate like git):
```

Comment:
Usually I'm not picky with spelling, but this is docs so I guess I will be.  "use less" is one word - "useless".